stocks/dark_pool_shorts/pos.py

```diff
@@ -0,0 +1,110 @@
+import discord
+import config_discordbot as cfg
+from helpers import pagination
+
+from gamestonk_terminal.stocks.dark_pool_shorts import stockgrid_model
+
+
+async def pos_command(ctx, arg, arg2):
+
+    # Help
+    if arg == "-h":
+        dark_pool_sort = {
+            "sv": "Short Vol. (1M)",
+            "sv_pct": "Short Vol. %",
+            "nsv": "Net Short Vol. (1M)",
+            "nsv_dollar": "Net Short Vol. ($100M)",
+            "dpp": "DP Position (1M)",
+            "dpp_dollar": "DP Position ($1B)",
+        }
+
+        help_txt = "Get dark pool short positions. [Source: Stockgrid]\n"
+
+        possible_args = ""
+        for k, v in dark_pool_sort.items():
+            possible_args += f"\n{k}: {v}"
+
+        help_txt += "\nPossible argments:\n"
+        help_txt += "<NUM> Number of top tickers to show. Default: 10\n"
+        help_txt += "<SORT> Field for which to sort by. Default: dpp_dollar\n"
+        help_txt += f"The choices are:{possible_args}"
+
+        embed = discord.Embed(
+            title="Stocks: [Stockgrid] Dark Pool Short Position HELP",
+            description=help_txt,
+            colour=cfg.COLOR,
+        )
+        embed.set_author(
+            name=cfg.AUTHOR_NAME,
+            icon_url=cfg.AUTHOR_ICON_URL,
+        )
+
+        await ctx.send(embed=embed)
+
+    else:
+        # Select default
+        if not arg:
+            arg = "10"
+        if not arg2:
+            arg2 = "dpp_dollar"
+
```

Comment:
We need to find a way for the bot to recognize that for example "!stocks.dps.pos sv" is "sv" is sort (arg2) not a num (arg2). This would be something new and maybe something for another pull request.
```suggestion
if arg in dark_pool_shorts: # Needs dark_pool_shorts to always be defined
   arg2 = arg
   arg = 10
```

Reply:
So, I think that the only way we can do that is by somehow specify the argument we are changing. However, for our user case - chat bots - having a long command line and complex is not ideal.

Thus, we are better off deciding which argument is "priority" in each example.

Example in this case I swaped these because the "sort field" is more important than the number of stocks displayed.

Reply:
Yeah, you're right!